cluster-install/README.md

```diff
@@ -0,0 +1,4 @@
+This folder is where the Astra cluster install script will live.
+- `kustomization.yaml`: this is a temporary kustomization pulled in by the install 
+script to install the neptune CRDs. It will be removed once the operator is updated to
+install them via Helm.
```

Comment:
Could def use some suggestions/opinions on the folder name, or where else to host the unified installer script (note: this is what customers will be downloading).